src/DependencyInjection/Configuration.php

```diff
@@ -16,7 +16,7 @@ class Configuration implements ConfigurationInterface
 {
     public function getConfigTreeBuilder(): TreeBuilder
     {
-        $tree = new TreeBuilder('symfony_request_id');
+        $tree = new TreeBuilder('symfony_trace');
```

Comment:
Looking at the recipes-contrib repository, we probably need to prefix this. Can you make this `dr_symfony_trace`?